src/core/client/stream/tabs/Comments/Stream/Subscriptions/CommentEnteredSubscription.ts

```diff
@@ -51,6 +52,11 @@ function updateForNewestFirst(
 
   // QUESTION: connection here was undefined when using the RR review/question tabs
   connection.setLinkedRecords(linked.concat(commentsEdge), "viewNewEdges");
+
+  const local = store.get(LOCAL_ID);
+  if (local) {
+    local.setValue(linked.length, "viewNewCount");
```

Comment:
This needs to be `linked.length + 1` to account for the new `commentsEdge`

Reply:
Fixed!